guides/micronaut-data-mongodb-synchronous/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "title": "Access a MongoDB database with Micronaut Data MongoDB",
+  "intro": "Learn how to access a MongoDB database with Micronaut Data and the MongoDB Sync driver.",
+  "authors": ["Tim Yates"],
+  "tags": ["database", "micronaut-data", "mongodb"],
+  "category": "Data Access",
+  "buildTools": ["gradle"],
+  "publicationDate": "2022-03-23",
+  "languages": ["java", "groovy"],
+  "apps": [
+    {
+      "name": "default",
+      "features": ["data-mongodb", "mongo-sync", "testcontainers"]
```

Comment:
this should not include `mongo-sync` feature?. Only `data-mongo` feature. 

<img width="817" alt="Screenshot 2022-04-07 at 13 26 13" src="https://user-images.githubusercontent.com/864788/162188460-e042bb80-5938-4d13-bc73-95d1a066cda9.png">

Reply:
How would we create a reactive mongo app then?

Which features would we apply?